tools/scoring/tests/parse_config_test.py

```diff
@@ -60,7 +60,7 @@ def testAppendTypes(self, mock_print):
     self.assertFalse(
         hasattr(
             list(self.parse.deserialized_files[PROPOSED].values())[0], 'type'))
-    deserialized_files_appended = self.parse.append_types(
+    deserialized_files_appended = self.parse._append_types(  # pylint: disable=protected-access
```

Comment:
make it public please and add docstring

Reply:
As above; should it be public if it is only being accessed as a shortcut in this test context and should otherwise be private?